libminifi/src/core/Processor.cpp

```diff
@@ -386,6 +379,39 @@ std::shared_ptr<Connectable> Processor::pickIncomingConnection() {
   return getNextIncomingConnectionImpl(rel_guard);
 }
 
+using annotation::input::EInputRequirement;
+
+void Processor::validateAnnotations() const {
```

Comment:
some unit tests could 'secure' this method

Reply:
Done.